resources/todomvc/architecture-examples/backbone/standalone/scripts/build.js

```diff
@@ -47,11 +48,13 @@ const build = async () => {
     // remove base paths from files to move
     for (let i = 0; i < filesToMove.length; i++) {
         const fileName = filesToMove[i].split("/").pop();
+        console.log(`${filesToMove[i]} -> ${fileName}`);
```

Comment:
Was these `console.log` for debugging?

Reply:
Yeah, all of them except `console.log("done!!");` which was already there.